validator/accounts/accounts.go

```diff
@@ -8,7 +8,11 @@ import (
 
 var log = logrus.WithField("prefix", "accounts")
 
-var msgKeymanagerNotSupported = "keymanager kind not supported: %s"
+var (
+	msgKeymanagerNotSupported = "keymanager kind not supported: %s"
+	// MsgCouldNotInitializeKeymanager informs about failed keymanager initialization
+	MsgCouldNotInitializeKeymanager = "could not initialize keymanager"
```

Comment:
Should be ErrCouldNotInitializeKeymanager

Reply:
Fixed in #8251